<?xml version="1.0"?>


		<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
		<!--                                                                       -->
		<!--                                                                       -->		
	    <!--                          ONE-STEP EXPLORATION                         -->
        <!--                       FOR GAS EMISSION MONITORING                     -->
		<!--                                                                       -->
		<!--                                                                       -->
		<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
		<!--                        @AsifArain, Nov 15, 2017                       -->
		<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->

<launch>

	<node pkg="plan_execution" type="robot_exploration_node" name="robot_plan_execution" output="screen">
	
    
        <!-- sensing parameters -->

	    <param name="field_of_view"	        value="180.0"/>
	    <param name="sensing_range"	        value="10.0"/> <!-- FIXME 8.25 is 10 meter in real -->
	    <param name="num_pan_sweeps"	    value="01"/> <!--10-->
	    <param name="num_tilt_sweeps"	    value="01"/> <!--3-->
	    <param name="sample_delay"	        value="0.1"/>
	
	
	    <!-- exploration type -->
	
	    <param name="file_plan"		    value="robot_plan.dat"/>
	    <param name="FilePath"		    value="$(find plan_execution)/logs/prismaforum5-02/two-step-exploration-detection/"/>
	
	
	    <!-- experiment/environment titles -->
		
	    <param name="experiment_title"	    value=  "prismaforum5-04"/>	
	    <param name="exploration_strategy"	value=  "two-step-exploration-detection"/>
	
	    <param name="cell_size_file"	value="prismaforum5_cellsize_conf.dat"/>
        <param name="robot_origin_file"	value="prismaforum5_origin_conf.dat"/> 
	

    </node>
</launch>


<!-- NOTES:

rosservice call /ptu_control/sweep "{min_pan: -45.0, max_pan: 45.0, min_tilt: -10.0, max_tilt: -10.0, n_pan: 1, n_tilt: 1, samp_delay: 0.1}"

For sensing range 10m: tilt angle= -5.1655
For sensing range 15m: tilt angle= -3.45
For sensing range 30m: tilt angle= -1.73




-->
